docs/reference/feast-cli-commands.md

```diff
@@ -165,22 +165,25 @@ List all registered permission
 feast permissions list
 
 Options:
-  --tags TEXT  Filter by tags (e.g. --tags 'key:value' --tags 'key:value,
-               key:value, ...'). Items return when ALL tags match.
-  --verbose
+  --tags TEXT    Filter by tags (e.g. --tags 'key:value' --tags 'key:value,
+                 key:value, ...'). Items return when ALL tags match.
+  -v, --verbose  Print the resources matching each configured permission
 ```
 
 ```text
 NAME                   TYPES            WITH_SUBCLASS    NAME_PATTERN           ACTIONS                             ROLES
```

Comment:
No need to display tags here?

Reply:
This is `feast permissions list` and IMO we could even avoid all the extras apart `types` and `actions` and keep the rest for the `describe` command that returns a YAML representation of the `Permission` instance (so, a full description). If we all agree, I can apply the change, otherwise we can keep as is and wait for feedbacks